test/OpenFeature.Tests/OpenFeatureEventTests.cs

```diff
@@ -257,8 +256,12 @@ public async Task API_Level_Event_Handlers_Should_Be_Executed_When_Other_Handler
             var testProvider = new TestProvider(fixture.Create<string>());
             await Api.Instance.SetProvider(testProvider);
 
-            failingEventHandler.Received().Invoke(Arg.Is<ProviderEventPayload>(payload => payload.ProviderName == testProvider.GetMetadata().Name));
-            eventHandler.Received().Invoke(Arg.Is<ProviderEventPayload>(payload => payload.ProviderName == testProvider.GetMetadata().Name));
```

Comment:
This was the flaky test (in 10 runs it failed twice locally). Notice it didn't have a sleep. I believe there was a race condition where only 1 of these 2 handlers had run by the time we hit the assertions.